C#/WinRT 1.1.2 causes multiple winrt.runtime.dll references to break ReadyToRun builds

example:
C:\Program Files (x86)\dotnet\sdk\5.0.103\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.Publish.targets(404,28): error MSB4094: "d:\NUGET_PACKAGES\microsoft.windows.cswinrt\1.1.2\lib\net5.0\WinRT.Runtime.dll;d:\NUGET_PACKAGES\microsoft.windows.cswinrt\1.1.2\lib\net5.0\WinRT.Runtime.dll;d:\NUGET_PACKAGES\microsoft.windows.cswinrt\1.1.2\lib\net5.0\WinRT.Runtime.dll" is an invalid value for the "CompilationEntry" parameter of the "RunReadyToRunCompiler" task. Multiple items cannot be passed into a parameter of type "Microsoft.Build.Framework.ITaskItem". [d:\git\winui\Samples\WinUICsDesktopSampleApp\WinUICsDesktopSampleApp\WinUICsDesktopSampleApp\WinUICsDesktopSampleApp.csproj]

workaround is to set PublishReadyToRun=false, which may not be feasible for all customers

--- a/nuget/Microsoft.Windows.CsWinRT.targets
+++ b/nuget/Microsoft.Windows.CsWinRT.targets
@@ -140,6 +140,6 @@ $(CsWinRTFilters)
   </Target>
 
   <Import Project="$(MSBuildThisFileDirectory)Microsoft.Windows.CsWinRT.Prerelease.targets" Condition="Exists('$(MSBuildThisFileDirectory)Microsoft.Windows.CsWinRT.Prerelease.targets')"/>
-  <Import Project="$(MSBuildThisFileDirectory)Microsoft.Windows.CsWinRT.Authoring.targets" Condition="Exists('$(MSBuildThisFileDirectory)Microsoft.Windows.CsWinRT.Authoring.targets')"/>
+  <Import Project="$(MSBuildThisFileDirectory)Microsoft.Windows.CsWinRT.Authoring.targets" Condition="'$(CsWinRTComponent)' == 'true'"/>
 
 </Project>

